src/plugins/i18n.js

```diff
@@ -0,0 +1,101 @@
+import Vue from 'vue';
```

Comment:
I'm wondering if pulling in i18n from dashboard should be a separate commit. There's a lot going on in both of these, and it would be easier to understand each on its own.